bot.py

```diff
@@ -51,54 +64,30 @@ def get_confirm_message(lang: str):
 
 
 def get_reply_payload(text: str, message_id: int) -> dict:
+    reply_data = {}
+    language = None
+    is_reply_needed = False
     r = random.randint(1, 100)
 
-    if 'филин' in text and any(confirm_str in text for confirm_str in ('подтверди', 'подтверждаешь')):
-        if r < 5:
-            return {'sticker': True}
-        return {
-            'text': get_confirm_message(RUS),
-            'reply_to_message_id': message_id
-        }
-    if 'owl' in text and 'confirm' in text:
-        return {
-            'text': get_confirm_message(ENG),
-            'reply_to_message_id': message_id
-        }
-    if 'der uhu' in text and 'bekräftig' in text:
-        return {
-            'text': get_confirm_message(GER),
-            'reply_to_message_id': message_id
-        }
-    if 'ზარნაშო' in text and 'დადასტურება' in text:
-        return {
-            'text': get_confirm_message(GEO),
-            'reply_to_message_id': message_id
-        }
-    if 'подтверди' in text:
-        if r < 5:
-            return {'sticker': True}
-        return {
-            'text': get_confirm_message(RUS),
-        }
-    if 'confirm' in text:
-        return {
-            'text': get_confirm_message(ENG),
-        }
-    if 'bekräftig' in text:
-        return {
-            'text': get_confirm_message(GER),
-        }
-    if 'დადასტურება' in text:
-        return {
-            'text': get_confirm_message(GER),
-        }
-
-    if r < 10:
-        return {
-            'text': CONFIRMS[RUS]['hoot']
-        }
-    return {}
+    for name in AGREE_NAMES.keys():
```

Comment:
Too many loops for me.
For more readability I think there is better to make functions for checking `is_reply_needed`, for languange detection etc.

Reply:
Will you do that please?

Reply:
Updated